apps/site/src/app/apply/sections/Form/AgeInformation.tsx

```diff
@@ -30,7 +30,7 @@ export default function AgeInformation() {
 			</div>
 
 			<SimpleRadio
-				name="minor-check"
+				name="minor_check"
```

Comment:
Question: is there a reason why you changed these to underscores? If I remember correctly, @taesungh is working on writing the code to process the application, so you might want to sync with him before changing the names.

Reply:
Yes, I will be updating this field name along with others as part of #93. I'm not sure if we should wait for my PR or if I should try to work with these new changes.

Reply:
If you haven't gotten too far with #93 yet (i.e. written the field names), we can merge this one first.

Reply:
I was testing to see if I could send the form in FastAPI (just a simple print statement), but for some reason having a hyphen in the parameters of the post request I was sending raised a compiler error, so I changed them all to underscores. 

Sorry, I should have thought about other people working with these names.

Reply:
No, that makes sense. Python can't process hyphens in names so I think that's why you're receiving those errors.

Reply:
No worries, this should probably have been noticed the first time around. I'm almost ready with the apply endpoint, but the file upload is also an important part I should test, so I can rebase after this is merged.